chia/wallet/puzzles/pool_member_innerpuz.clvm

```diff
@@ -16,13 +14,15 @@
   ; POOL_REWARD_PREFIX is network-specific data (mainnet vs testnet) that helps determine if a coin is a pool reward
   ; WAITINGROOM_PUZHASH is the puzzle_hash you'll go to when you iniate the leaving process
 
-  ; spend_type is: 0 for absorbing money, 1 to escape
+  ; spend_type is absorbing money if p1 is a cons box, spend_type is escape if p1 is an atom
+
+  ; p1 is (pool_reward_amount . pool_reward_height) if absorbing money
+  ; p1 is key_value_list if escaping
```

Comment:
I don't believe this is a good idea. I understand that `key_value_list` is a serialized blob; the intent was for it to be a clvm list of key-value pairs (which would make it a list also).

I also understand that's not how it currently work. My intuition is that we will regret doing it this way (which means, I don't have anything specific to point to yet).

Reply:
I absolutely agree, and am changing this back.